### PR TITLE
Document that PowerShell scripts are run detached

### LINKED ISSUE #7274: Add chechbox "Refresh after" in script parameters
I use script:
"Is powershell" checkbox enabled
Command: git checkout -B "master" "origin/master";git pull --rebase --progress "origin";git submodule update --init --recursive;exit;

Every time it completes I must click Refresh button manually. Please add ability to do this automatically.

### PR DESCRIPTION
**Pull request by mstv**
## Proposed changes

- add comment why the RevisionGrid cannot be refreshed for PowerShell scripts
- improve code readability

## Test methodology 

- no functional change

## Test environment(s) 

- Git Extensions 3.3.0
- Build 697478fd79be9d4b7d4518c1a8bf3583008d2f5f
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4018.0
- DPI 96dpi (no scaling)

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by mstv** [comment]
Feel free to adopt this PR and implement waiting for PowerShell to finish.

**Comment by mstv** [comment]
Reverted the single function change, added only a comment and kept the readability improvements.

**Comment by mstv** [comment]
@msftbot merge in one day

**Comment by mstv** [comment]
@msftbot merge in 1 day

**Comment by gerhardol** [comment]
@mstv 
This should be rebased before merged, it is hard to follow the history with merges like this.
(I prefer -ff-only merges to get a history you can understand.)

**Comment by mstv** [comment]
Yes, I have rebased now, Gerhard.
